sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosPartitionKeyTests.java

```diff
@@ -53,7 +53,7 @@ public CosmosPartitionKeyTests(CosmosClientBuilder clientBuilder) {
     }
 
     @BeforeClass(groups = { "simple" }, timeOut = SETUP_TIMEOUT)
-    public void beforeClass() throws URISyntaxException, IOException {
+    public void before_CosmosPartitionKeyTests() throws URISyntaxException, IOException {
```

Comment:
any reason for renaming this method to `before_CosmosPartitionKeyTests`
If we are changing the test name pattern for beforeClass we should be consistent everywhere.

on the other test class you changed to `before_CosmosSyncStoredProcTest`
If this change is needed could you be consistent on "test" vs "tests"